### ISSUE TITLE
Connecting to DBus in namespace not possible

### ISSUE THREAD

**Issue by jonasdn** [jonasdn/nsntrace#4]
Example: 

** (epiphany:18019): WARNING **: Couldn't connect to accessibility bus: Failed to connect to socket /tmp/dbus-zVCTcw3GtQ: Connection refused

**Comment by jonasdn** [comment]
Possibly could any of the work-arounds in this stackexchange work for us?https://unix.stackexchange.com/questions/184964/connect-with-d-bus-in-a-network-namespace
